tests/main.test.ts

```diff
@@ -271,7 +271,7 @@ it(`shouldn't allow using regular Yarn commands on npm-configured projects`, asy
 
     await expect(runCli(cwd, [`yarn`, `--version`])).resolves.toMatchObject({
       exitCode: 1,
-      stderr: ``,
+      stderr: expect.stringContaining(`This project is configured to use npm`),
```

Comment:
A regex seems simpler, no?
```suggestion
      stderr: /This project is configured to use npm/,
```